src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,50 @@
+$blueAccent: #00acdc;
```

Comment:
move variables to separate file _variables.scss and https://github.com/import it in main.scss